appeals/web/src/server/appeals/appeal-details/service-user/service-user.service.js

```diff
@@ -0,0 +1,24 @@
+/**
+ *
+ * @param {import('got').Got} apiClient
+ * @param {string} appealId
+ * @param {number} serviceUserId
+ * @param {string} userType
+ * @param {{firstName: string; lastName: string; emailAddress: string;}} data
+ * @returns {Promise<{}>}
+ */
+export function updateServiceUser(apiClient, appealId, serviceUserId, userType, data) {
+	const formattedServiceUser = {
```

Comment:
Very minor, this could be defined inline in the `patch` call below - either way is fine though